02week/pigLatin.js

```diff
@@ -9,10 +9,24 @@ const rl = readline.createInterface({
 
 
 const pigLatin = (word)  => {
+// Your code here
+  let vowels = "AEIOUaeiou";
+  let pigged = "";
+    for( let i = 0; i < word.length; i++) {
+        if(vowels.includes(word[0])) {
```

Comment:
this check does not need to happen inside the loop, since you are only checking the first letter.